include/cling/Utils/Casting.h

```diff
@@ -0,0 +1,56 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author: Roman Zulak
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+
+#ifndef CLING_UTILS_CASTING_H
+#define CLING_UTILS_CASTING_H
+
+namespace cling {
+  namespace utils {
+
+    ///\brief Legally cast a pointer to a function to void*.
+    ///
+    /// \param [in] The function to cast
+    ///
+    /// \return A void* of the functions address.
+    ///
+    template <class T>
+    void* FunctionToVoid(T *fPtr) {
```

Comment:
Can we throw in `Ptr`? "Void" Without "Ptr" sounds weird, but then we also need it for Function, so I guess `FunctionPtrToVoidPtr`? Or `FunPtrToVoidPtr` is fine, too.